ts/components/NoticeBanner.tsx

```diff
@@ -53,18 +53,18 @@ export const NoticeBanner = (props: NoticeBannerProps) => {
 
 const StyledGroupInviteBanner = styled(Flex)`
   position: relative;
-  background-color: var(--orange-color);
   color: var(--black-color);
+  background-color: var(--orange-color);
   font-size: var(--font-size-sm);
   padding: var(--margins-xs) var(--margins-lg);
   text-align: center;
   flex-shrink: 0;
 
   // when part a a dialog, invert it and make it narrower (as the dialog grows to make it fit)
   ${StyledRootDialog} & {
-    background-color: unset;
-    color: var(--text-primary-color);
     max-width: 300px;
+    color: var(--warning-color);
```

Comment:
Should we not have an is warning prop as well? Just check if we are in a dialog to using the warn color seems to generic.

Reply:
I think that's fine. We could have a prop to invert the color, but then we'd also need one for isInDialog for the maxWidth.
We will tweak this when/if we ever need to